lib/ocfl/object/directory_builder.rb

```diff
@@ -34,27 +40,33 @@ def create_object_directory
       # @return [Directory]
       def save
         version_path = object_root + "v1"
-        FileUtils.mkdir_p(version_path) unless version_path.exist? # in case no files were added
-
-        FileUtils.touch(object_root + "0=ocfl_object_1.1")
+        FileUtils.mkdir_p(version_path)
```

Comment:
I don't think we need to check whether the version path exists since we're using `mkdir_p`